RULES_EXPAND

```diff
@@ -68,43 +69,43 @@ $(1):	$$(DIR_LIST) $$($(1)_DEP_LIST)
 $(BUILD_TOP)/iocBoot/$(1)/Makefile:      $(IOC_APPL_TOP)/iocBoot/templates/Makefile
 	@echo Expanding $$@
 	@$(EXPAND) -c $(1).cfg $$(<) $$@ IOCNAME=$(1) TOP=$(BUILD_TOP_ABS) IOCTOP=$(IOC_APPL_TOP)
-	@chmod ug+w $$@
+	@-chmod ug+w $$@
 
 # Expand $(BUILD_TOP)/iocBoot/$(1)/st.cmd
 $(BUILD_TOP)/iocBoot/$(1)/st.cmd:      $(IOC_APPL_TOP)/iocBoot/templates/st.cmd
 	@echo Expanding $$@
 	@$(EXPAND) -c $(1).cfg $$(<) $$@ IOCNAME=$(1) TOP=$(BUILD_TOP_ABS) IOCTOP=$(IOC_APPL_TOP)
-	@chmod ug+w,a+x $$@
+	@-chmod ug+w,a+x $$@
```

Comment:
Make syntax reminder (for me): `-` prefix means to ignore the exit status of the executed command. That is, `chmod` won't fail these make rules.

Reply:
Yep.  All of these '-' are certainly correct.  I have to admit, I'm a little weaker when it comes to the .PHONY and some of the other changes.  This makefile was Bruce's baby.  expand.py was mine.